SomeFile.java

```diff
@@ -0,0 +1,19 @@
+package com.test.common;
+
+public class SomeFile {
+
+  private static String message = "some text";
+
+  public void SomeMethod() {
+    otherMethod(3);
+    anotherMethod(true, false);
+    System.out.println(message);
+  }
+
+  private void otherMethod(int unused) {
+  }
+
+  private boolean anotherMethod(boolean couldBe, boolean maybeNot) {
```

Comment:
A private method that does not reference the enclosing instance can be static.

[Show suggested change](url)

Reply:
private static void otherMethod(int unused) {}